files/ansible/templates/ompi_hello_world.ini.j2

```diff
@@ -28,6 +42,19 @@ plugin = Shell
 
 # Use our built ompi install
 command = mpicc /home/test/mtt/samples/python/mpi_hello_world.c -o mpi_hello_world 
+modules = openmpi3
+modules_unload = openmpi4/4.1.0
```

Comment:
Isn't this going to conflict with the module template above?

Reply:
No, here it unloads our module (so our snapshot MPI stack) and loads the OHPC one.